WooCommerce/Classes/ViewRelated/Blaze/AdDestination/BlazeAdDestinationSettingView.swift

```diff
@@ -8,72 +8,81 @@ struct BlazeAdDestinationSettingView: View {
 
     typealias DestinationType = BlazeAdDestinationSettingViewModel.DestinationURLType
 
+    @State private var isShowingAddParameterView = false
+
     init(viewModel: BlazeAdDestinationSettingViewModel) {
         self.viewModel = viewModel
     }
 
     var body: some View {
         NavigationView {
             VStack(alignment: .leading, spacing: 0) {
-                // URL destination section
-                VStack(spacing: 0) {
-                    sectionHeading(title: Localization.destinationUrlHeading)
-                    destinationItem(title: Localization.productURLLabel,
-                                    subtitle: String(format: Localization.destinationUrlSubtitle, viewModel.productURL),
-                                    type: DestinationType.product,
-                                    showBottomDivider: true)
-
-                    destinationItem(title: Localization.siteHomeLabel,
-                                    subtitle: String(format: Localization.destinationUrlSubtitle, viewModel.homeURL),
-                                    type: DestinationType.home)
-                }
-                .padding(.bottom, Layout.sectionVerticalSpacing)
+                List {
+                    Section {
+                        destinationItem(title: Localization.productURLLabel,
+                                        subtitle: String(format: Localization.destinationUrlSubtitle, viewModel.productURL),
+                                        type: DestinationType.product)
+
+                        destinationItem(title: Localization.siteHomeLabel,
+                                        subtitle: String(format: Localization.destinationUrlSubtitle, viewModel.homeURL),
+                                        type: DestinationType.home)
+                    } header: {
+                        Text(Localization.destinationUrlHeading)
+                    }
 
-                // URL parameters section
-                VStack {
-                    sectionHeading(title: Localization.urlParametersHeading)
 
-                    VStack {
-                        ForEach(viewModel.parameters, id: \.self) { parameter in
-                            parameterItem(itemName: parameter.key)
+                    Section {
+                        ForEach(viewModel.parameters.indices, id: \.self) { index in
```

Comment:
> Make BlazeAdURLParameter conform to Identifiable. Since each key is unique, you can use that for each object's id.

A concern is that duplicate keys in query strings are not prohibited, and the WHATWG URL Standard mentions about handling query strings with duplicate keys. See this SO discussion: https://stackoverflow.com/a/70264621

I suppose I'll have to add let id=UUID() then?

Reply:
I see, let's use UUID then.

Reply:
Added changes in 

https://github.com/woocommerce/woocommerce-ios/pull/11642/commits/df03a0895d2233195bf261cf692d2cce2954416f